release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ocp-4-16-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: release-notes
+
+Do not add or edit release notes here. Edit release notes directly in the branch
+that they are relevant for.
+
+Release note changes should be added/edited in their own PR.
+
+This file is here to allow builds to work.
```

Comment:
This statement feels unnecessary to me, but feel free to disagree. :slightly_smiling_face:

Reply:
Ill leave it in the file is missed which can easily happen when operating on an enterprise branch and forgetting to update `main`.